pitr/ddl.go

```diff
@@ -120,6 +120,7 @@ func (d *DDLHandle) ExecuteDDL(schema string, ddl string) error {
 	}
 
 	if _, err := d.db.Exec(ddl); err != nil {
+		fmt.Println(err)
```

Comment:
useless line?

Reply:
forget to address, I will delete it next pr